lib/crystalball/map_storage/yaml_storage.rb

```diff
@@ -33,7 +33,7 @@ def read_files(path)
 
           paths.map do |file|
             metadata, *example_groups = file.read.split("---\n").reject(&:empty?).map do |yaml|
-              YAML.safe_load(yaml, [Symbol])
+              YAML.safe_load(yaml, permitted_classes: [Symbol])
```

Comment:
This version: `YAML.safe_load(yaml, [Symbol])` was deprecated by Psych, then removed in version 4.